_posts/2016-09-25-Neonto-try-their-hands-at-react.mdown

```diff
@@ -0,0 +1,20 @@
+---
+layout: post
+author: keatontaylor
+title: New Policies for iOS | Webkit Blog 
+categories: link
+custom_type: link
+link_url: https://webkit.org/blog/6784/new-video-policies-for-ios/
+tags: [Design, Development, tools]
+date: 2016-09-25 2:54
+---
+
+[Neonto](https://www.neonto.com/) has just announced the impending release of a new design tool. Neonto Studio allows users to "Build real native mobile apps better and faster." Their new offering - [React Studio](https://reactstudio.com/) - is a design tool that creates single page React.js web apps. 
+
+From their site: 
+
+> Exported code conforms to latest React+Webpack best practices. Image and video assets are properly resized and compressed.
+
+This is at once interesting and suspect to me. Is it because I still have a bad taste in my mout from DreamWeaver and iWeb? Maybe. Is it because I write code and am skeptical that machine output is as good as what a human can write? For sure. The thing is though, Neonto seems to have a pretty decent track record doing this kind of thing. 
+
+I'm doubtful this tool will be able to yield high-functioning and scalable web apps. That said, they definitely have my attention. I think this could be a great gateway into understanding React in a more holistic way - especially for designers. 
```

Comment:
Remove "I think" from the sentence here, we already know it's your thought:

> I'm doubtful this tool will be able to yield high-functioning and scalable web apps. That said, they definitely have my attention. It could be a great gateway into understanding React in a more holistic way—especially for designers.